tests/unit/fetch-config.spec.ts

```diff
@@ -43,7 +43,6 @@ xdescribe('Fetch Config', () => {
     await fetchConfig()
 
     // verify data
-    expect(sessionStorage.getItem('ACCOUNT_ID')).toBe('2288')
```

Comment:
Somewhere, we should have a test to verify that the account id is correctly parsed and used, ie:
- in BreadcrumbResources.ts
- in navigate.ts

Maybe app.spec.ts would be a good place? I'd be OK with a new describe section in there just to keep it separate from everything else.

Reply:
I'm hoping for some tests that verify that both places this value is used, it's properly parsed from the current account object and they build the correct output string (for breadcrumb or navigate).

Reply:
Ah, OK, I see now. Looks good.